### PR TITLE
Making paths an optional param in RunOptions, defaulting to `.`

### PR DESCRIPTION
**Pull request by carakessler**
Giving paths a default value in RunOptions, used by CheckupTaskRunner

### CODE REVIEW

File: packages/core/src/types/cli.ts [hunk 1]
```diff
@@ -9,7 +9,7 @@ export type RunOptions = {
   excludePaths?: string[];
   groups?: string[];
   listTasks?: boolean;
-  paths: string[];
+  paths?: string[];
```
Comment (scalvert) [inline packages/core/src/types/cli.ts:12]: Sorry, why are we making `paths` optional? We should always require a path, right?

**Review by scalvert — CHANGES_REQUESTED** [review]

File: packages/core/src/types/cli.ts (hunk 1 above)
Comment (carakessler) [reply packages/core/src/types/cli.ts:12]: i thought we discussed the path being optional and defaulting it to `.`?

Comment (scalvert) [reply packages/core/src/types/cli.ts:12]: Just defaulting it, not the optional bit (unless I missed that). 

What does it mean to operate on no paths?

Comment (scalvert) [reply packages/core/src/types/cli.ts:12]: Also I think we need paths to be an array, in this case of one element.

Comment (scalvert) [reply packages/core/src/types/cli.ts:12]: We resolved this offline.

**Review by scalvert — APPROVED** [review]

**Pull request merged**